OpenUpSA/wazimap-ng-ui
# show login modal for private profiles

## Description

For profiles marked private a login modal should be shown
401 in request log which I think is supposed to trigger login prompt
"Another login already sent" in the console log
no login prompt
## Related Issue


## How to test it locally

Find a private profile, if none, change the settings for a profile on stage, and the modal should show up requiring you to log in.

## Screenshots
Login modal without close button

![Screenshot from 2021-07-14 14-26-32](https://user-images.githubusercontent.com/6994982/125614674-e40e146d-8f88-45d8-8199-03a49936dc47.png)

### Removed forgot password label
![Screenshot from 2021-07-15 12-47-07](https://user-images.githubusercontent.com/6994982/125767733-1de302c8-5c20-43ae-9e27-3e78f03ec9fc.png)
